web/src/lib/components/asset-viewer/asset-viewer.svelte

```diff
@@ -188,7 +190,12 @@
     }
   }
 
+  const handleFullScreenChange = () => {
+    isFullScreen = document.fullscreenElement !== null;
```

Comment:
https://svelte.dev/docs/special-elements#svelte-document

Reply:
Damn, I always forget about that one, thanks !